subworkflows/local/quality_control/main.nf

```diff
@@ -2,17 +2,17 @@
 // Perform quality control
 //
 
-include { NACHO_QC } from '../../../modules/local/nacho/qc/main.nf'
+include { NACHO_QC } from '../../../modules/nf-core/nacho/qc/main'
 
 workflow QUALITY_CONTROL {
     take:
-    counts      // channel: path(rcc files)
-    samplesheet // file: /path/to/samplesheet.csv
+    counts      // channel: [ meta, [rcc files] ]
+    samplesheet // channel: [ meta, path/to/samplesheet.csv ]
 
     main:
     NACHO_QC ( counts, samplesheet )
 
     emit:
-    nacho_qc_multiqc_metrics = NACHO_QC.out.nacho_qc_multiqc_metrics // channel: [ .png and .txt mqc files ]
-    versions                 = NACHO_QC.out.versions                 // channel: [ versions.yml ]
+    nacho_qc_multiqc_metrics = NACHO_QC.out.nacho_qc_png.map{it[1]}.mix(NACHO_QC.out.nacho_qc_txt.map{it[1]}) // channel: [ .png and .txt mqc files ]
```

Comment:
Better to use join here instead of mix, this way all only the outputs that have the same meta will be merged together. This doesn't impact the pipeline now because we don't allow to run more than one sample at a time, and all of the files get mixed together.